devenv/datasources.yaml

```diff
@@ -311,3 +311,13 @@ datasources:
           spanEndTimeShift: '-5m'
           customQuery: true
           query: '{filename="/var/log/grafana/grafana.log"} |="$${__span.traceId}"'
+      tracesToProfiles:
+        datasourceUid: gdev-pyroscope
+        profileTypeId: "process_cpu:cpu:nanoseconds:cpu:nanoseconds"
```

Comment:
scratch that, i see it in the description now

> nly CPU profiling is fully supported

Reply:
You have to pick one as I understand it. Once you've navigated to the CPU profile you can select memory. But so far I think only trace propagation is supported for CPU profiles as I mentioned in description. Need to investigate to understand better.

Reply:
From https://grafana.com/docs/pyroscope/next/configure-client/trace-span-profiles/go-span-profiles/

> NOTE
>
> - Only CPU profiling is supported.
> - Because of how sampling profilers work, spans shorter than the sample interval may not be captured. Go CPU profiler probes stack traces 100 times per second, meaning that spans shorter than 10ms may not be captured.